src/main/resources/META-INF/plugin.xml

```diff
@@ -260,8 +260,8 @@
                 text="Accept Edit"
                 description="Accept the fixup">
             <!-- Intended as ctrl shift PLUS but that is not recognized by IntelliJ. -->
-            <keyboard-shortcut first-keystroke="ctrl shift EQUALS" keymap="$default"/>
-            <keyboard-shortcut first-keystroke="ctrl shift EQUALS" keymap="Mac OS X 10.5+"/>
+            <keyboard-shortcut first-keystroke="ctrl shift EQUALS" keymap="$default" replace-all="true"/>
```

Comment:
Can we do that for al the binding in the `plugin.xml`?
If we will change any of those shortcuts in future we will run into the same issue.

Reply:
I have applied this change to all actions which have more than one shortcut. I'm not sure if we should apply this to the other actions. After all, there must be some reasons why the API developers haven't made this behaviour the default.